### ISSUE TITLE
ui-select-search has invalid ARIA (aria-expanded) attribute

### ISSUE THREAD

**Issue by elduderino87** [angular-ui/ui-select#2009]
When using the `selectize` theme with `multiple='true'`, the `aria-selected` attribute is applied to the hidden `<input type='search' class='ui-select-search....'`

This is according to the google chrome accessibility extension developer tool.

![capture](https://user-images.githubusercontent.com/3092346/26922803-89936e48-4c83-11e7-8227-0c9d5faa4bc0.PNG)

My workaround is to remove the (line 2426) in `select.js`. is this a valid workaround? It seems to get rid of the accessibility error at least....I'm happy to submit a PR.

**Comment by Jefiozie** [comment]
I don't think this is a valid workaround as you are removing from the template cache (correct me if I'm wrong).  

I'm not a aria expert so maybe you can have a look if this problem is only in newer version of the library? Or have some alternative ideas to fix this?

_Sent from my Samsung SM-A310F using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by mfrankwork** [comment]
The same accessibility error [[This element has an unsupported ARIA attribute](https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_aria_10)] occurs with the single select element in the `bootstrap` theme.  I believe `aria-expanded="true"` should be removed from the single select template (`select.tpl.html`) for the `bootstrap` and `select2` themes.

**Comment by irbian** [comment]
I´m also interested on this issue
